src/tools/vfs/vfs_main.c

```diff
@@ -10,9 +10,370 @@
 
 #include "vfs_daemon.h"
 
+#include <sys/wait.h>
+#include <limits.h>
+#include <assert.h>
+#include <ctype.h>
 
-int main(int argc, char **argv)
+
+vfs_opts_t g_opts = {
+    .action         = VFS_DAEMON_ACTION_START,
+    .foreground     = 0,
+    .verbose        = 0,
+    .mountpoint_dir = VFS_DEFAULT_MOUNTPOINT_DIR,
+    .mount_opts     = "",
+    .sock_path      = NULL
+};
+
+const char *vfs_action_names[] = {
+    [UCS_VFS_SOCK_ACTION_STOP]  = "stop",
```

Comment:
remove `UCS_` prefix for stop and mount actions

Reply:
why? it's defined in src/**ucs**/vfs/sock/vfs_sock.h